packages/devtools_app/lib/src/ui/hover.dart

```diff
@@ -215,23 +223,93 @@ class HoverCard {
   }
 }
 
+/// Ensures that only one [HoverCard] is ever displayed at a time.
+class HoverCardTooltipController {
+  /// The card that is currenty being displayed
+  HoverCard? _currentHoverCard;
+
+  /// Sets [hoverCard] as the most recently displayed [HoverCard].
+  ///
+  /// This methods sets a unique [hoverCard.freshnessToken], as a way to track whether
```

Comment:
this comment seems obsolete. Also, maybe rename to 'show' since 'set' is kind of a loaded word? (setters)